presto-server/src/main/provisio/presto.xml

```diff
@@ -116,7 +116,7 @@
         </artifact>
     </artifactSet>
 
-    <artifactSet to="plugin/teradata-functions">
+    <artifactSet to="plugin/td-functions">
         <artifact id="${project.groupId}:presto-teradata-functions:zip:${project.version}">
```

Comment:
This was because the path was too long, we probably don't want to change the artifact name such that it's inconsistent with facebook/presto